From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: quh4gko8 <88831734+quh4gko8@users.noreply.github.com>
Date: Wed, 17 Jun 2026 00:24:55 +0000
Subject: [PATCH] Partially revert "add support for other language content
 filtering dependent on language settings"

This reverts commit 686b64a37986690583517d16efb7b2d4e77dcd7f.
---
 DEPS | 200 -----------------------------------------------------------
 1 file changed, 200 deletions(-)

diff --git a/DEPS b/DEPS
index 221b15acb030b..d0ad75c72150f 100644
--- a/DEPS
+++ b/DEPS
@@ -4997,206 +4997,6 @@ hooks = [
               'https://easylist.to/easylist/easylist.txt',
               'https://easylist.to/easylist/easyprivacy.txt']
   },
-  {
-    'name': 'fetch_filter_lists_arabic',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_arabic.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/liste_ar.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_bulgaria',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_bulgaria.txt',
-              '--urls',
-              'http://stanev.org/abp/adblock_bg.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_spanish',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_spanish.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/easylistspanish.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_french',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_french.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/liste_fr.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_germany',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_germany.txt',
-              '--urls',
-              'https://easylist.to/easylistgermany/easylistgermany.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_hebrew',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_hebrew.txt',
-              '--urls',
-              'https://raw.githubusercontent.com/easylist/EasyListHebrew/master/EasyListHebrew.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_indian',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_indian.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/indianlist.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_indonesia',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_indonesia.txt',
-              '--urls',
-              'https://raw.githubusercontent.com/heradhis/indonesianadblockrules/master/subscriptions/abpindo.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_italy',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_italy.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/easylistitaly.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_korean',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_korean.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/koreanlist.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_lithuanian',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_lithuanian.txt',
-              '--urls',
-              'https://raw.githubusercontent.com/EasyList-Lithuania/easylist_lithuania/master/easylistlithuania.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_latvian',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_latvian.txt',
-              '--urls',
-              'https://raw.githubusercontent.com/Latvian-List/adblock-latvian/master/lists/latvian-list.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_dutch',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_dutch.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/easylistdutch.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_nordic',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_nordic.txt',
-              '--urls',
-              'https://raw.githubusercontent.com/DandelionSprout/adfilt/master/NorwegianExperimentalList%20alternate%20versions/NordicFiltersABP-Inclusion.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_polish',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_polish.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/easylistpolish.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_portuguese',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_portuguese.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/easylistportuguese.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_romanian',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_romanian.txt',
-              '--urls',
-              'https://zoso.ro/pages/rolist.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_russian',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_russian.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/ruadlist.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_vietnam',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_vietnam.txt',
-              '--urls',
-              'https://abpvn.com/filter/abpvn-IPl6HE.txt']
-  },
-  {
-    'name': 'fetch_filter_lists_china',
-    'condition': 'checkout_android',
-    'action': ['python3',
-              'src/vanadium/android_config/filter_lists/filter_list_download.py',
-              '--output',
-              'src/vanadium/android_config/filter_lists/filter_lists_easylist_china.txt',
-              '--urls',
-              'https://easylist-downloads.adblockplus.org/easylistchina.txt']
-  },
 
 ]
 
